.github/workflows/labeler.yml

```diff
@@ -14,16 +15,13 @@ jobs:
   triage:
     runs-on: ubuntu-latest
     steps:
+
     - name: Add agent-java label
-      uses: actions/github-script@v7
+      uses: AlexanderWert/issue-labeler@32be4a3c3d8f009c2741af471994337c34b4cb6f  # v2.3
       with:
-        script: |
-          github.rest.issues.addLabels({
-            issue_number: context.issue.number,
-            owner: context.repo.owner,
-            repo: context.repo.repo,
-            labels: ["agent-java"]
-          })
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
+        configuration-path: .github/java-label.yml
+        enable-versioned-regex: 0
```

Comment:
[for reviewer] this part failed, so switching to the same pattern we have elsewhere for example in the android agent seem to work. Also, I wonder if we could/should replace the current version as it relies on Alex repository.